diff --git a/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c b/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c
index 5195807f1f3d..3d7118b75328 100644
--- a/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c
+++ b/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c
@@ -2572,7 +2572,8 @@ static int dw_hdmi_connector_get_modes(struct drm_connector *connector)
                                             connector);
        struct edid *edid;
        struct drm_display_mode *mode;
-       const u8 def_modes[6] = {4, 16, 31, 19, 17, 2};
+       //const u8 def_modes[6] = {4, 16, 31, 19, 17, 2};
+       const u8 def_modes[1] = {76};                                                         // LWH-changed: 76对应drm_edid.c 中 edid_cea_modes的76 - 1920x1080@60Hz
        struct drm_display_info *info = &connector->display_info;
        struct hdr_static_metadata *metedata =
                        &connector->display_info.hdmi.hdr_panel_metadata;
@@ -2581,7 +2582,8 @@ static int dw_hdmi_connector_get_modes(struct drm_connector *connector)
        if (!hdmi->ddc)
                return 0;
 
-       edid = drm_get_edid(connector, hdmi->ddc);
+       //edid = drm_get_edid(connector, hdmi->ddc);
+       edid = NULL;
        if (edid) {
                dev_dbg(hdmi->dev, "got edid: width[%d] x height[%d]\n",
                        edid->width_cm, edid->height_cm);
						
						
						
/media/colorlight/nvme/A2K/kernel/drivers/gpu/drm/drm_edid.c
 /* 76 - 1920x1080@60Hz */
        { DRM_MODE("1920x1080", DRM_MODE_TYPE_DRIVER, 148500, 1920, 2008,
                   2052, 2200, 0, 1080, 1084, 1089, 1125, 0,
                   DRM_MODE_FLAG_PHSYNC | DRM_MODE_FLAG_PVSYNC),
          .vrefresh = 60, .picture_aspect_ratio = HDMI_PICTURE_ASPECT_64_27, },
						
						
						
						
						
						
						
						
						
			